### PR TITLE
Added parameter for configuring timeout for /evaluate.

### PR DESCRIPTION
**Pull request by jnegara**
Added support for parameter `TABPY_EVALUATE_TIMEOUT` in the server config.

When `/evaluate` is called with some script, if that script takes longer than the number of seconds specified by this parameter the script is terminated and the client received a `TimeoutError`.

If `TABPY_EVALUATE_TIMEOUT` is set to a non-float value, or is not present then the timeout length defaults to 30s.

This should close issue #118.

### CODE REVIEW

File: docs/server-config.md [hunk 1]
```diff
@@ -192,3 +193,12 @@ arg1, _arg2):
 No passwords are logged.
 
 NOTE the request context details are logged with INFO level.
+
+## Custom Script Timeout
+
+By default, all custom scripts execute through `POST /evaluate` may run for up
```
Comment (sbabayan) [inline docs/server-config.md]: executed*

File: docs/server-config.md [hunk 2]
```diff
@@ -192,3 +193,12 @@ arg1, _arg2):
...
+to 30.0 s before being terminated. To configure this timeout, uncomment
+`TABPY_EVALUATE_TIMEOUT = 30` in the default config under
+`tabpy-server/tabpy_server/common/default.conf` and replace `30` with the float
+value of your choice representing the timeout time in seconds, or add an such
```
Comment (sbabayan) [inline docs/server-config.md]: or add such*

File: tests/integration/test_custom_evaluate_timeout.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        self._run_test(payload)
+
+    def _run_test(self, payload):
+        conn = self._get_connection()
+        conn.request('POST', '/evaluate', payload, self.headers)
+        res = conn.getresponse()
+        res_bytes = res.read().decode('utf-8')
+
+        self.assertEqual(500, res.status)
+        print('res_bytes: ' + res_bytes)
```
Comment (sbabayan) [inline tests/integration/test_custom_evaluate_timeout.py]: Why does the integration test need to print?

File: tabpy-server/tabpy_server/handlers/evaluation_plane_handler.py [hunk 4]
```diff
@@ -5,22 +5,24 @@
...
+        self.timeout = timeout
 
     def query(self, name, *args, **kwargs):
         url = f'http://localhost:{self.port}/query/{name}'
         self.logger.log(logging.DEBUG, f'Querying {url}...')
         internal_data = {'data': args or kwargs}
         data = json.dumps(internal_data)
         headers = {'content-type': 'application/json'}
+
+        print(f'timeout = {self.timeout}')
```
Comment (0golovatyi) [inline tabpy-server/tabpy_server/handlers/evaluation_plane_handler.py]: remove this - debugging artifact?

File: tests/integration/test_custom_evaluate_timeout.py (hunk 3 above)
Comment (jnegara) [reply tests/integration/test_custom_evaluate_timeout.py]: it doesn't, good catch

File: tabpy-server/tabpy_server/handlers/evaluation_plane_handler.py (hunk 4 above)
Comment (jnegara) [reply tabpy-server/tabpy_server/handlers/evaluation_plane_handler.py]: yup

**Pull request merged**